assembl/static2/js/app/components/administration/discussion/personalizeInterface/load.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import type { ApolloClient } from 'react-apollo';
+
+import DiscussionPreferences from '../../../../graphql/DiscussionPreferencesQuery.graphql';
+import type { PersonalizeInterfaceValues } from './types.flow';
+
+export const load = async (client: ApolloClient, fetchPolicy: FetchPolicy) => {
+  const { data } = await client.query({
+    query: DiscussionPreferences,
+    fetchPolicy: fetchPolicy
+  });
+  return data;
+};
+
+export function postLoadFormat(data: DiscussionPreferencesQuery): PersonalizeInterfaceValues {
+  const { discussionPreferences } = data;
+  return {
+    title: discussionPreferences && discussionPreferences.tabTitle,
+    favicon: discussionPreferences && discussionPreferences.favicon
```

Comment:
`discussionPreferences ? discussionPreferences.favicon : null` for the same reason (to avoid to pass `undefined` value even if in this case I think that the `FileUploader` doesn't care)